nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AbstractDatabaseFetchProcessor.java

```diff
@@ -231,7 +243,9 @@ public void setup(final ProcessContext context) {
 
         // Try to fill the columnTypeMap with the types of the desired max-value columns
         final DBCPService dbcpService = context.getProperty(DBCP_SERVICE).asControllerService(DBCPService.class);
-        final String tableName = context.getProperty(TABLE_NAME).evaluateAttributeExpressions().getValue();
+        final String propTableName = context.getProperty(TABLE_NAME).evaluateAttributeExpressions().getValue();
+        final String tableName = org.apache.commons.lang3.StringUtils.isEmpty(propTableName) ? ARBITRARY_SQL_TABLE_NAME : propTableName;
```

Comment:
Why the fully-qualified StringUtils class? If we have both (NiFi and Commons Lang), can we get rid of one?

Reply:
Just some copy/paste action from another chunk of code.